verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -154,6 +154,7 @@ static std::vector<absl::Status> ExtractFiles(
   if (absl::GetFlag(FLAGS_printextraction)) {
     std::cout << file_list_facts_tree << std::endl;
   }
+  LOG(INFO) << file_list_facts_tree << std::endl;
```

Comment:
This will be pretty large. Maybe use VLOG?

Reply:
Ah sorry I always use it to debug because I don't have VLOG.